CHANGELOG.md

```diff
@@ -26,3 +19,4 @@ v0.0.1
 * Added (optional) replacement for "Add block" button, showing blocks in a categorized popup
 * Fixed PHP-CS-Fixer config
 * Added integration of a liipMonitor dashboard
+* Add overriding of lexik maintenance to lock and unlock by site
```

Comment:
hhm wrong update here. This line should be in master part with the lines you removed